src/appleseed.python/CMakeLists.txt

```diff
@@ -107,8 +108,12 @@ set (sources
 list (APPEND appleseed.python_sources
     ${sources}
 )
+set (blender_progressive_tile_callback_sources
+    bindblenderprogressivetilecallback.cpp
```

Comment:
If this file is in the same folder as the other files, include it in `${sources}` in the previous `set ()` command.

Reply:
I want to compile it only when building the python3 bindings.